src/app/app.component.html

```diff
@@ -1,38 +1,35 @@
 <div class="title">Video App</div>
 
-<!-- <div id="player"> </div> -->
-<!-- <iframe id="ytplayer" type="text/html" width="720" height="405" src="https://www.youtube.com/embed/M7lc1UVf-VE"
-  frameborder="0" allowfullscreen> -->
-
 <div class="search">
-  <button (click)="add()">Dodaj Film</button>
+  <button class="add-movie" (click)="add()">Dodaj Film</button>
   <input [(ngModel)]="newTask">
 </div>
 
-<div class="film">
-  <h1>Moje filmy</h1>
-  <button class="filterfilm">Ulubione filmy</button>
-  <button class="sortfilm">Sortuj</button>
-</div>
+<div class="movie">
+  <div>
+    <p>Moje filmy: </p>
+    <button class="filtr-movie">Ulubione filmy</button>
+    <button class="delete-videos">Usuń wszystkie filmy</button>
 
+  </div>
+  <div><br>
+    <app-my-movies></app-my-movies>
+    <!-- <p>Sortuj: </p> -->
+    <!-- <button class="oldest-movies">Najstarsze filmy</button>
+    <button class="added-movies"> Ostatnio dodane filmy</button> -->
+  </div>
+</div>
 <section class="container">
   <div class="view">
     <button class="tiles">Kafelki</button>
-    <button class="listview">Lista</button>
+    <button class="list-view">Lista</button>
   </div>
   <div class="display">
-    <ng-container *ngIf="tasksList.length > 0; else noTask">
+    <ng-container *ngIf="films.length > 0; else noTask">
```

Comment:
powiedziałbym że 0 jest falsy i wystarczy napisać ngIf=films.length, ale sprawdź czy to działa

Reply:
noi  noTask zmień na noFilm